### PR TITLE
Add DevToolbox plugin 🛠

### PR DESCRIPTION
**Pull request by theankitgaurav**
Add nifty utilities for everyday developer use in the menu bar. 😊

### CODE REVIEW

File: Tools/dev-toolbox.1h.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+# <xbar.author.github>theankitgaurav</xbar.author.github>
+# <xbar.desc>Handy menu bar app to serve as swiss-army knife for devs.</xbar.desc>
+#  <xbar.image>https://user-images.githubusercontent.com/9376908/185213187-c6ad3f67-217c-47a4-9283-10d92b1f2aa9.png</xbar.image>
+# <xbar.dependencies>shell,jq,sed</xbar.dependencies>
+# <xbar.abouturl>https://github.com/theankitgaurav/dev-toolbox</xbar.abouturl>
+
+# Variables as preferences of the app:
+#  <xbar.var>select(APP_ICON="⚙️"): App icon to be shown in the menu app. [⚙️,🛠,DevToolBox]</xbar.var>
+
+export PATH="/usr/local/bin:/usr/bin:$PATH"
```
Comment (sprak3000) [inline Tools/dev-toolbox.1h.sh:14]: Might want to add the new M1 silicon path for Homebrew:

```suggestion
export PATH="/opt/homebrew/bin:/usr/local/bin:/usr/bin:$PATH"
```

**Review by sprak3000 — COMMENTED** [review]
@theankitgaurav 

I've been trying to use this but don't seem to get anything happening when I use the menu links. Could you outline how this is supposed to work?

**Comment by theankitgaurav** [comment]
Hi @sprak3000 , I dont have an m1 machine to test the modification. So I have made the commit as suggestion.
Does that work?

**Comment by sprak3000** [comment]
@theankitgaurav 

Yes, that is fine. I can't seem to get the plugin to do anything in normal use. I use the menu items, and nothing ends up on my clipboard. Alternate mode works though. Going to merge as is, but this might need a bit more work to get it running fully.

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**